sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResource.java

```diff
@@ -299,12 +299,7 @@ public Response createRow(
     }
     logger.info("createRow(): try to call backend with CQL of '{}'", cql);
 
-    QueryOuterClass.QueryParameters params =
-        QueryOuterClass.QueryParameters.newBuilder()
-            .setConsistency(
-                QueryOuterClass.ConsistencyValue.newBuilder()
-                    .setValue(QueryOuterClass.Consistency.LOCAL_QUORUM))
-            .build();
+    QueryOuterClass.QueryParameters params = parametersForLocalQuorum();
```

Comment:
Does the REST API default to LOCAL_QUORUM for all queries?

Reply:
Seems to default everywhere I have seen in v1 resource methods. I don't think there is a way for caller to change it.

Reply:
Yes, we default to `LOCAL_QUORUM` but there have been some requests to make CL configurable.